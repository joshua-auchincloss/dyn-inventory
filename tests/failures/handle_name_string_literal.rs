// handle_name must be an identifier
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    MyTrait: MyStruct<T> {
        data: u64,
        t: T,
    };
    handle_name = "HandleThing",
);

fn main() {}
