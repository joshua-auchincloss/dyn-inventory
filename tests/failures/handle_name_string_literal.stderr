error: expected identifier
  --> tests/failures/handle_name_string_literal.rs:18:19
   |
18 |     handle_name = "HandleThing",
   |                   ^^^^^^^^^^^^^
